pkg/networkmanager/network-interface-members.jsx

```diff
@@ -109,10 +103,7 @@ export const NetworkInterfaceMembers = ({
                                     onClick={syn_click(model, () => {
                                         with_checkpoint(
                                             model,
-                                            function () {
-                                                return (free_member_connection(member_con)
-                                                        .fail(show_unexpected_error));
-                                            },
+                                            () => free_member_connection(member_con).catch(show_unexpected_error),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19120-20230721-193803-d973134a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)